docs/installation.rst

```diff
@@ -98,27 +98,26 @@ GEOS discovery (runtime)
 ------------------------
 
 PyGEOS is dynamically linked to GEOS. This means that the same GEOS library that was used
-during PyGEOS compilation is required on your system. Make sure that the dynamic linker paths are
-set such that the libraries are found.
+during PyGEOS compilation is required on your system at runtime. When using pygeos that was distributed
+as a binary wheel or through conda,this is automatically the case and you can stop reading.
 
-When using a binary wheel, conda, the Anaconda shell, or the OSGeo4W shell, this is automatically
-the case and you can stop reading. In other cases this can be tricky, especially if you have
-multiple GEOS installations next to each other. For PyGEOS this can be solved by prepending
-the GEOS path before you start Python.
+In other cases this can be tricky, especially if you have multiple GEOS installations next
+to each other. We only include some guidelines here to address this issue as this document is
+not intended as a general guide of shared library finding.
 
-On Linux::
-
-    $ export LD_LIBRARY_PATH=/path/to/geos/lib:$LD_LIBRARY_PATH
-    $ sudo ldconfig  # refresh dynamic linker cache
-
-On OSX::
+There are in general four ways of making Python aware of the location of shared library:
```

Comment:
On linux, if GEOS was automatically found during compile time, it also won't be found during runtime?

(I am totally no expert here, but just thinking about what I do myself: for example for development, I am building pygeos from source (and thus not using a binary wheel or conda package), and I don't think I need to do any of the 4 options below to have it working)

Reply:
On Linux the libraries must be on the “ld” path at runtime. You can check if that is the case using “ldconfig -p | grep geos”

If you don’t have to do any of the 4 options, probably one of them is already the case. Maybe geos’ cmake puts something in your /usr/local/lib?

Reply:
In my case it's a conda environment, so GEOS is located in the `/lib` directory of the environment (so I suppose the equivalent of `/usr/local/lib` for the environment).

It's just that I would find the current paragraph a bit confusing. Maybe more clearly say that one of those actions should only be used if not working out of the box (I now incorrectly got the impression that I would _need_ to do one of those 4 actions to get it working).

Reply:
I worked a bit on the paragraph, I hope it is more clear now